lang/LangSource/SC_TerminalClient.cpp

```diff
@@ -166,6 +168,11 @@ bool SC_TerminalClient::parseOptions(int& argc, char**& argv, Options& opt)
 			case 'r':
 				opt.mCallRun = true;
 				break;
+			case 'V':
+				fprintf(stdout, "sclang %s\n", SC_VersionString().c_str());
+				quit(0);
+				return false;
+				break;
```

Comment:
"return false" and then "break"? this looks odd. break not needed, at least, since unreachable